client/scripts/models/Account.ts

```diff
@@ -97,12 +96,8 @@ abstract class Account<C extends Coin> {
     if (!signature && (this.seed || this.mnemonic || this.chainBase === ChainBase.NEAR)) {
       // construct signature from private key
       signature = await this.signMessage(`${this._validationToken}\n`);
-    } else if (signature) {
-      const withoutNewline = !(await this.isValidSignature(this._validationToken, signature));
```

Comment:
Any reason we're removing client-side signature checking?

Reply:
Just to simplify and reduce cost of maintenance, it was getting tricky to maintain so multiple copies of verification code for each type of chain.